src/components/dev-hub/card-list.js

```diff
@@ -41,71 +41,76 @@ const getThumbnailUrl = media => {
         : media.thumbnailUrl;
 };
 
-// TODO: Update this component to take one array of items and map to cards based
-// on the item type
+const sortCardsByDate = contentList =>
+    contentList.sort((a, b) => {
```

Comment:
```suggestion
    contentList.sort((a, b) => new Date(b.publishDate || b.pubdate) - new Date(a.publishDate || a.pubdate))
```

Since we are comparing with `-` we can shorten this to one line using a quick subtraction comparison between dates!

This also would not work for articles, since these support `pubdate` and not `publishDate`, so `||` chaining should do the trick.